src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -2172,7 +2172,7 @@ public void EventHandler_OnElement_WithString()
 
             // Act
             var generated = CompileToCSharp(@"
-<input @onclick=""foo"" />");
+<input onclick=""foo"" />");
```

Comment:
Does this test make sense anymore?

Reply:
no, delete

Reply:
This test does make sense but could be renamed. This is a JS event handler now 👍.

We should continue to have tests that the following work:
```razor
<input onclick=""foo"" />
<input onclick=""@foo"" />
```

These are both JS event handlers, and in the latter case it should be an expression that's going to get ToStringed() and added as a JS event handler 👍

Reply:
Wait whaa, we understand non-directive attribute event handlers and do things special? Don't we just add this stuff as content and let the dom evaluate?

Reply:
We're both right. 
- This is a normal DOM attribute
- I still want us to have *some* tests for this case.